modules/atomic-widgets/assets/js/editor/commands-internal/create-style.js

```diff
@@ -0,0 +1,72 @@
+import { __ } from '@wordpress/i18n';
+
+/**
+ * @typedef {import('elementor/assets/dev/js/editor/container/container')} Container
+ */
+export class CreateStyle extends $e.modules.editor.CommandContainerInternalBase {
+	validateArgs( args ) {
+		this.requireContainer( args );
+
+		this.requireArgumentConstructor( 'bind', String, args );
+	}
+
+	randomId( containerId ) {
+		return `s-${ containerId }-${ elementorCommon.helpers.getUniqueId() }`;
+	}
+
+	apply( args ) {
+		const { container, styleDefId, bind } = args;
+		const oldStyles = container.model.get( 'styles' ) || {};
+
+		/* Translators: 1: container label, 2: number of old styles */
+		const label = sprintf( __( '%1$s Style %2$s', 'elementor' ), container.label, Object.keys( oldStyles ).length + 1 );
+
+		const newStyle = {
+			id: styleDefId ?? this.randomId( container.id ),
+			label,
+			type: 'class',
+			variants: [],
+		};
+
+		const $$type = 'classes'; // TODO: Style Transformer should be used here.
```

Comment:
```suggestion
		const propType = 'classes';
```

you can also move it outside the class and call it something like `PROP_TYPE_CLASSES`